install-devstack-xen.sh

```diff
@@ -487,9 +495,9 @@ ENABLE_TENANT_VLANS=True
 Q_ML2_TENANT_NETWORK_TYPE=vlan
 ML2_VLAN_RANGES="physnet1:1100:1200"
 
-PUB_IP=172.24.4.1
-SUBNETPOOL_PREFIX_V4=192.168.10.0/24
-NETWORK_GATEWAY=192.168.10.1
+#PUB_IP=172.24.4.1
```

Comment:
suggest delete these lines directly since there are such default definition in devstack